docs/src/about_guide.md

```diff
@@ -12,7 +12,8 @@ Start typing...
 
 Type | Description
 --- | ---
-Hot | Shows *trending* posts, based on the score, and the most recent comment time.
+Active | Shows *trending* posts, based on the score, and the most recent comment time.
+Hot | Shows *trending* posts, based on the score, and the post creation time.
```

Comment:
Ah so the main difference is that one is based on comment activity, and the other on post activity. Its hard to figure out from these sentences that are almost identical. How about:

> Trending sort based on votes and comment activity

Cause this is supposed to be a simple explanation for new users. Detailed explanation is in another document (and that could go more in depth about the details of each sorting method.

Reply:
most recent comment time is simple enough.

Reply:
Maybe just put that part first?

Reply:
I updated, is that one okay?

Reply:
Yes, sounds better to me.